src/current/cockroachcloud/export-metrics-standard.md

```diff
@@ -0,0 +1,328 @@
+
+{{site.data.alerts.callout_success}}
+CockroachDB Standard clusters use Cloud Console instead of DB Console, and DB Console is disabled. To export metrics from a CockroachDB {{ site.data.products.core }} cluster, refer to [Monitoring and Alerting](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/monitoring-and-alerting) instead of this page.
+{{site.data.alerts.end}}
+
+Exporting metrics to AWS CloudWatch is only available on CockroachDB Standard clusters which are hosted on AWS. Metrics export to Datadog is supported on all CockroachDB Standard clusters.
+
+{{site.data.alerts.callout_info}}
+{% include_cached feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+<a id="the-metricexport-endpoint"></a>
+
+## `metricexport` endpoint
+
+To configure and manage metrics export for your CockroachDB Standard cluster, use the `metricexport` endpoint appropriate for your desired cloud metrics sink:
+
+Cloud metrics sink | Metrics export endpoint
+------------------ | ----------------------------------------------------
+AWS Cloudwatch     | `https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/metricexport/cloudwatch`
+Datadog            | `https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/metricexport/datadog`
+
+Access to the `metricexport` endpoints requires a valid CockroachDB {{ site.data.products.cloud }} [service account]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) with the appropriate permissions (`admin` privilege or Cluster Admin role).
+
+The following methods are available for use with the `metricexport` endpoints, and require the listed service account permissions:
+
+Method | Required permissions | Description
+--- | --- | ---
+`GET` | `ADMIN`, `EDIT`, or `READ` | Returns the current status of the metrics export configuration.
+`POST` | `ADMIN` or `EDIT` | Enables metrics export, or updates an existing metrics export configuration.
+`DELETE` | `ADMIN` | Disables metrics export, halting all metrics export to AWS CloudWatch or Datadog.
+
+See [Service accounts]({% link cockroachcloud/managing-access.md %}#manage-service-accounts) for instructions on configuring a service account with these required permissions.
+
+## Enable metrics export
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="aws-metrics-export">AWS CloudWatch</button>
+  <button class="filter-button" data-scope="datadog-metrics-export">Datadog</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="aws-metrics-export">
+
+Exporting metrics to AWS CloudWatch is only available on CockroachDB Standard clusters which are hosted on AWS. If your CockroachDB Standard cluster is hosted on GCP or Azure, you can [export metrics to Datadog](export-metrics.html?filters=datadog-metrics-export) instead.
```

Comment:
We don't offer Azure standard clusters today. Not sure if we want to include that. Note that there are a couple of Azure references in the doc.

Reply:
removed references to Azure.